Source/UETorch/Private/UETorch.cpp

```diff
@@ -973,4 +973,65 @@ extern "C" bool DestroyActor(AActor* object) {
 	}
 	object->K2_DestroyActor();
 	return true;
+}
+
+/**
+ * Functions to control pawns
+ */
+
+extern "C" bool SimpleMoveToLocation(UObject* _this, AActor* object, float x, float y, float z) {
```

Comment:
with this function you don't need to indicate the location at each time, just the end points, and then if you add a Nav Mesh Bounds Volume (https://docs.unrealengine.com/latest/INT/Engine/AI/BehaviorTrees/QuickStart/2/) it is going to find a path even if there are some obstacles along the way